src/sentry/tsdb/base.py

```diff
@@ -12,6 +15,12 @@
 ONE_DAY = ONE_HOUR * 24
 
 
+@dataclass
+class IncrMultiOptions(TypedDict):
```

Comment:
`@dataclass` and `TypedDict` is probably not correct?

Reply:
Yup, nice catch.